Dockerfile

```diff
@@ -116,7 +116,8 @@ RUN wget https://github.com/samtools/samtools/archive/0.1.19.tar.gz && \
     rm -r samtools-0.1.19
 
 # Install the latest R
-RUN add-apt-repository 'deb https://cloud.r-project.org/bin/linux/ubuntu xenial-cran35/' && \
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 51716619E084DAB9 && \
```

Comment:
@ljmciver Just confirmed that both 119 and 121 is needed for a successful build.